app/src/main/res/xml/appfilter.xml

```diff
@@ -1890,6 +1900,9 @@
     <!-- MiXplorer -->
     <item component="ComponentInfo{com.mixplorer/com.mixplorer.activities.BrowseActivity}" drawable="mixplorer" />
 
+    <!-- MMT-365 -->
+    <item component="ComponentInfo{com.mmt.applications.chronometer/com.mmt.applications.chronometer.ScreenSplash}" drawable="mmt-365"/>
```

Comment:
Can you rename this icon without the dash?

Reply:
renamed